app/containers/Vaults/migrationWhitelist.json

```diff
@@ -33,7 +33,7 @@
   },
   {
     "symbol": "WETH",
-    "token": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+    "token": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
```

Comment:
I requested this change. We are lucky in that it does not seem to matter but in general we should checksum addresses (the other addresses are checksummed)